src/overview/actions.js

```diff
@@ -258,11 +265,19 @@ export const showTags = index => (dispatch, getState) => {
 
 export const filterTag = tag => (dispatch, getState) => {
     const query = selectors.query(getState())
+    const filterTags = selectors.filterTags(getState())
```

Comment:
Call `getState()` once at the start of the thunk then reuse the state from that invocation for each selector. Else you're deriving from the state at two different times (pretty sure it will never be an issue to actually produce weird results)